src/mixins/filing-template-mixin.ts

```diff
@@ -496,6 +498,14 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.STAFF
 
     // store Correction Information
+    if (
+      this.isCorrectedRegistration ||
+      this.isCorrectedChangeReg ||
+      this.isCorrectionFiling
```

Comment:
These should match the conditions when you save (build) the filing above.

You don't need to check if this is a correction filing -- this method is specific to corrections already.

Reply:
Done

Reply:
I think this still isn't right. See the comment I made on line 187.

Reply:
Okay I found the issue I was checking for the correction filing type before setting the correction filing type. I fixed it now.